SC_redesign/Crop_and_Soil/soil/layer_data.py

```diff
@@ -135,3 +233,8 @@ def water_factor(self):
         else:
             return (self.saturation_content - self.soil_water_content) / (
                     self.saturation_content - self.field_capacity_content)
+
+    @property
+    def silt_clay_content(self):
```

Comment:
```suggestion
    def silt_clay_ratio(self):
```
I think this is a better name for this quantity actually represents.